snmp/datadog_checks/snmp/data/default_profiles/netgear-readynas.yaml

```diff
@@ -103,12 +104,13 @@ metrics:
           name: netgear.readynasos.volumeStatus
         tag: netgear_readynasos_volume_status
         mapping:
-          1: redundant
-          2: unprotected
+          1: online
+          2: offline
           3: degraded
-          4: dead
-          5: inactive
+          4: faulted
+          5: removed
```

Comment:
Seems to not match the definition here https://github.com/DataDog/snmp-mibs/blob/main/mibs/READYNASOS-MIB#L361-L374

```

		volumeStatus OBJECT-TYPE
			SYNTAX DisplayString
			ACCESS read-only
			STATUS mandatory
			DESCRIPTION
				"The severity of the volume.
				Possible values:
				1: REDUNDANT
				2: UNPROTECTED
                                3: DEGRADED
                                4: DEAD
			        5: INACTIVE
                                6: UNKNOWN"
			::= { volumeEntry 4 }
```

Reply:
all references for this OID point to the READYDATAOS MIB where the [mapping](https://github.com/DataDog/snmp-mibs/blob/main/mibs/READYDATAOS-MIB#L377-L391) is the new one
[example](https://github.com/cisco-kusanagi/mibs.snmplabs.com/blob/48bb18954e5db428383114ff13b372b9d0cccfb2/json/READYDATAOS-MIB.json#L473) or [here](https://github.com/cisco-kusanagi/mibs.snmplabs.com/blob/48bb18954e5db428383114ff13b372b9d0cccfb2/json/READYDATAOS-MIB.json#L473)
I reached out to @iammowgoud about the origin of the READYNASOS MIB

Reply:
reverted this back to the old mapping and limited this profile to just redaynasos devices (refer to the latest version)